src/components/rangeslider/draw.js

```diff
@@ -504,7 +504,7 @@ function drawRangePlot(rangeSlider, gd, axisOpts, opts) {
 
         var plotinfo = {
             id: id,
-            plotgroup: plotgroup,
+            plotgroup: [undefined, plotgroup],
```

Comment:
Could you please explain why the undefined is needed?

Reply:
It's because `makeSubplotLayerAbove` and `makeSubplotLayerBelow` expect plotinfo.plotgroup to be an array now with the bg group at index 0 and fg group at index 1.
But the RangePlot apparently doesn't contain anything that would now be drawn in the bg layer, so the corresponding plotgroup is undefined.

It was easiest for me to do it like this. Alternatively, there could also be a `plotinfo.plotgroupFg` and `plotinfo.plotgroupBg`?

Reply:
No. Let's keep it simple and please add a comment instead.